include/af/array.h

```diff
@@ -1580,7 +1580,7 @@ extern "C" {
        \ingroup method_mat
        @{
 
-       Get the use count of `af_array`
+       Get the reference count of \ref af_array
```

Comment:
:+1:

Reply:
I can't find this in my current commit, so I think it is fixed.